test/remote/gateways/remote_paymentez_test.rb

```diff
@@ -255,6 +256,16 @@ def test_failed_capture
     assert_equal 'The modification of the amount is not supported by carrier', response.message
   end
 
+  def test_successful_capture_with_otp
+    @options[:vat] = 0.1
+    response = @gateway.purchase(@amount, @otp_card, @options)
+    assert_equal 'pending', response.params['transaction']['status']
```

Comment:
Thank you for the helpful note about the Ecuador fixtures. This test passes when I use the Ecuador fixtures, but a lot of other tests fail.

Therefore I think it would be a good idea to define a second set of fixtures in this test file (maybe called `paymentez_ecuador`). Then we could create an `ecuador_gateway` in this file and use that for this test. We could use our standard Brazil fixtures for the rest of the tests.

See [here](https://github.com/activemerchant/active_merchant/blob/master/test/remote/gateways/remote_mercado_pago_test.rb#L7) for an example of what I mean.